src/main/webapp/web.xml

```diff
@@ -30,48 +31,105 @@
 
     <servlet-mapping>
         <servlet-name>registration</servlet-name>
-        <url-pattern>/user/registration</url-pattern>
+        <url-pattern>/registration</url-pattern>
     </servlet-mapping>
 
     <servlet>
         <servlet-name>deleteUser</servlet-name>
-        <servlet-class>com.internet.shop.controllers.DeleteUserController</servlet-class>
+        <servlet-class>com.internet.shop.controllers.user.DeleteUserController</servlet-class>
     </servlet>
     <servlet-mapping>
         <servlet-name>deleteUser</servlet-name>
-        <url-pattern>/user/delete</url-pattern>
+        <url-pattern>/users/delete</url-pattern>
     </servlet-mapping>
     <servlet>
-        <servlet-name>allProducts</servlet-name>
-        <servlet-class>com.internet.shop.controllers.GetAllProductsController</servlet-class>
+        <servlet-name>buyProduct</servlet-name>
+        <servlet-class>com.internet.shop.controllers.cart.AddProductToCartController</servlet-class>
     </servlet>
     <servlet-mapping>
-        <servlet-name>allProducts</servlet-name>
-        <url-pattern>/product/all</url-pattern>
+        <servlet-name>buyProduct</servlet-name>
+        <url-pattern>/shopping-cart/products/add</url-pattern>
     </servlet-mapping>
+
     <servlet>
-        <servlet-name>buyProduct</servlet-name>
-        <servlet-class>com.internet.shop.controllers.AddProductToCartController</servlet-class>
+        <servlet-name>deleteProduct</servlet-name>
+        <servlet-class>com.internet.shop.controllers.cart.DeleteProductFromCartController</servlet-class>
     </servlet>
+
     <servlet-mapping>
-        <servlet-name>buyProduct</servlet-name>
-        <url-pattern>/shopping-cart/products/add</url-pattern>
+        <servlet-name>deleteProduct</servlet-name>
+        <url-pattern>/products/delete</url-pattern>
```

Comment:
```suggestion
        <url-pattern>/shopping-cart/products/delete</url-pattern>
```
add a controller with mapping `/products/delete` for deletion of product from storage (admin does this)